docs/source/developer/extension_dev.rst

```diff
@@ -679,3 +681,36 @@ Currently supported package managers are:
 
 - ``pip``
 - ``conda``
+
+
+
+Shipping Packages
+^^^^^^^^^^^^^^^^^
+Most extensions are single JavaScript packages, and can be shipped on npmjs.org.
+This makes them discoverable by the JupyterLab extension manager, provided they
+have the ``"extension"`` metadata in their ``package.json``.  If the package also
```

Comment:
Unless someone made some changes, this isn't strictly correct. The mechanism the extension manager is having a `keyword` with the value `jupyterlab-extension`.

Reply:
Fixed in 1830388